src/components/content/Cards/ContentCard.tsx

```diff
@@ -0,0 +1,112 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { Box, Typography, useTheme } from '@mui/material'
+import { CardChip, StatusVariants } from './CardChip'
+
+export interface ContentCardProps {
+    title: string
+    chipText: string
+    bpnHeading?: string
+    bpnDetails?: string
```

Comment:
please remember the shared components should not contain any business specific details, so the prefix `bpn` for business partner number is not good here. Let's simply call it "heading" and "detail"